README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I'm going to say as a custom helper method, since helpers actually write html using Rails / the project's conventions. Then I'd call the helper method in the view files.
```

Comment:
:+1: exactly! we need a helper to generate specific html code. How do you mean elaborate?